### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by SMOLKEYS** [DeltaNedas/modding-toolbox#5]
~~goddamnit delta~~

```
Mindustry has crashed. How unfortunate.
Version: pre-alpha build 135
OS: Linux xnull (aarch64)
Java Version: 0
Mods: betamindy:1.07
[#b59e72]Demo of Chaos Itself[], dev-mode:2.3.2, hackustry:4, modding-toolbox:3.7.3, modding-tools:0.1, new-horizon:1.10.2.1 Beta, newconsole:1.3, newcontrols:1.0.4, notepad:2.0.0, override-lib:1.0.1, pictologic:1.5.7, prog-mats:21.2, test-utils:27, testarea:1.4.8, testers:0.25, time-control:0.2, ui-lib:2.16.6, unit-factory:2.4.2, world-edit:1, yellow-java:0.32

> EcmaError: 'TypeError: init is not a function, it is object. (elements#26)'
ScriptRuntime.constructError: 3741
ScriptRuntime.constructError: 3721
ScriptRuntime.typeError: 3750
ScriptRuntime.typeError2: 3766
ScriptRuntime.notFunctionError: 3813
ScriptRuntime.nameOrFunction: 1727
ScriptRuntime.getNameFunctionAndThis: 2120
OptRuntime.callName: 49
elements_27._c_anonymous_2: 26
elements_27.call: 47
ArrowFunction.call: 33
OptRuntime.call1: 23
uieditor_26._c_anonymous_1: 32
uieditor_26.call: 41
ArrowFunction.call: 33
OptRuntime.callProp0: 71
uieditor_26._c_anonymous_19: 199
uieditor_26.call: 149
ContextFactory.doTopCall: 328
ScriptRuntime.doTopCall: 3114
uieditor_26.call: 18
ArrowFunction.call: 33
InterfaceAdapter.invokeImpl: 141
InterfaceAdapter.lambda$invoke$0: 92
InterfaceAdapter.$r8$lambda$eCbvUJKud9uRuKkOr-KOpaMfYY8: 0
InterfaceAdapter$$ExternalSyntheticLambda0.run: 13
Context.call: 450
ContextFactory.call: 428
InterfaceAdapter.invoke: 92
VMBridge.lambda$newInterfaceProxy$0: 142
VMBridge.$r8$lambda$2rbCXLLvnIIB7GRsWozxBZ9kMPs: 0
VMBridge$$ExternalSyntheticLambda0.invoke: 11
Proxy.invoke: 1006
$Proxy2.run: -1
Element.lambda$clicked$2: 893
Element.$r8$lambda$HTGgyzI_sft57MMc4a-IcKQNKts: 0
Element$$ExternalSyntheticLambda2.get: 4
Element$4.clicked: 902
ClickListener.touchUp: 77
InputListener.handle: 31
Scene.touchUp: 366
InputMultiplexer.touchUp: 136
AndroidInput.processEvents: 294
AndroidGraphics.onDrawFrame: 367
GLSurfaceView$GLThread.guardedRun: 1571
GLSurfaceView$GLThread.run: 1270
```

**Comment by SMOLKEYS** [comment]
(caused by clicking the export button in the ui editor)

**Comment by deltanedas** [comment]
maybe fixed in 3.7.4 i have no idea

**Comment by SMOLKEYS** [comment]
> maybe fixed in 3.7.4 i have no idea

delta please

```
> EcmaError: 'TypeError: Cannot find function indexOf in object 
elem => "table.add(\"" + escapestr(elem.properties.text) + "\")"
. (elements#30)'
ScriptRuntime.constructError: 3741
ScriptRuntime.constructError: 3721
ScriptRuntime.typeError: 3750
ScriptRuntime.typeError2: 3766
ScriptRuntime.notFunctionError: 3829
ScriptRuntime.getPropFunctionAndThisHelper: 2192
ScriptRuntime.getPropFunctionAndThis: 2175
elements_27._c_anonymous_2: 30
elements_27.call: 47
ArrowFunction.call: 33
OptRuntime.call1: 23
uieditor_26._c_anonymous_1: 32
uieditor_26.call: 41
ArrowFunction.call: 33
OptRuntime.callProp0: 71
uieditor_26._c_anonymous_19: 199
uieditor_26.call: 149
ContextFactory.doTopCall: 328
ScriptRuntime.doTopCall: 3114
uieditor_26.call: 18
ArrowFunction.call: 33
InterfaceAdapter.invokeImpl: 141
InterfaceAdapter.lambda$invoke$0: 92
InterfaceAdapter.$r8$lambda$eCbvUJKud9uRuKkOr-KOpaMfYY8: 0
InterfaceAdapter$$ExternalSyntheticLambda0.run: 13
Context.call: 450
ContextFactory.call: 428
InterfaceAdapter.invoke: 92
VMBridge.lambda$newInterfaceProxy$0: 142
VMBridge.$r8$lambda$2rbCXLLvnIIB7GRsWozxBZ9kMPs: 0
VMBridge$$ExternalSyntheticLambda0.invoke: 11
Proxy.invoke: 1006
$Proxy2.run: -1
Element.lambda$clicked$2: 893
Element.$r8$lambda$HTGgyzI_sft57MMc4a-IcKQNKts: 0
Element$$ExternalSyntheticLambda2.get: 4
Element$4.clicked: 902
ClickListener.touchUp: 77
InputListener.handle: 31
Scene.touchUp: 366
InputMultiplexer.touchUp: 136
AndroidInput.processEvents: 294
AndroidGraphics.onDrawFrame: 367
GLSurfaceView$GLThread.guardedRun: 1571
GLSurfaceView$GLThread.run: 1270
```

and another one if you decide to export it while it contains nothing

```
> EcmaError: 'TypeError: Cannot find function show in object [object Object]. (uieditor#36)'
ScriptRuntime.constructError: 3741
ScriptRuntime.constructError: 3721
ScriptRuntime.typeError: 3750
ScriptRuntime.typeError2: 3766
ScriptRuntime.notFunctionError: 3829
ScriptRuntime.getPropFunctionAndThisHelper: 2192
ScriptRuntime.getPropFunctionAndThis: 2175
OptRuntime.callProp0: 69
uieditor_26._c_anonymous_1: 36
uieditor_26.call: 41
ArrowFunction.call: 33
OptRuntime.callProp0: 71
uieditor_26._c_anonymous_19: 199
uieditor_26.call: 149
ContextFactory.doTopCall: 328
ScriptRuntime.doTopCall: 3114
uieditor_26.call: 18
ArrowFunction.call: 33
InterfaceAdapter.invokeImpl: 141
InterfaceAdapter.lambda$invoke$0: 92
InterfaceAdapter.$r8$lambda$eCbvUJKud9uRuKkOr-KOpaMfYY8: 0
InterfaceAdapter$$ExternalSyntheticLambda0.run: 13
Context.call: 450
ContextFactory.call: 428
InterfaceAdapter.invoke: 92
VMBridge.lambda$newInterfaceProxy$0: 142
VMBridge.$r8$lambda$2rbCXLLvnIIB7GRsWozxBZ9kMPs: 0
VMBridge$$ExternalSyntheticLambda0.invoke: 11
Proxy.invoke: 1006
$Proxy2.run: -1
Element.lambda$clicked$2: 893
Element.$r8$lambda$HTGgyzI_sft57MMc4a-IcKQNKts: 0
Element$$ExternalSyntheticLambda2.get: 4
Element$4.clicked: 902
ClickListener.touchUp: 77
InputListener.handle: 31
Scene.touchUp: 366
InputMultiplexer.touchUp: 136
AndroidInput.processEvents: 294
AndroidGraphics.onDrawFrame: 367
GLSurfaceView$GLThread.guardedRun: 1571
GLSurfaceView$GLThread.run: 1270
```

**Comment by deltanedas** [comment]
stupid hack but whatever update
